lib/shared/config.rb

```diff
@@ -0,0 +1,48 @@
+require 'json'
+
+require_relative 'errors/file_not_found'
+
+module GoogleDrive
+  class Config
```

Comment:
Add

    #:nodoc:

at the end of this line to suppress generating RDoc for this internal class.